pydis_site/apps/content/models/tag.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+
+class Tag(models.Model):
+    """A tag from the python-discord server."""
+
+    last_updated = models.DateTimeField(
+        help_text="The date and time this data was last fetched.",
+        auto_now=True,
+    )
+    name = models.CharField(
+        help_text="The tag's name.",
+        primary_key=True,
+        max_length=50,
+    )
+    body = models.TextField(help_text="The content of the tag.")
+    url = models.URLField(help_text="The URL to this tag on GitHub.")
```

Comment:
does this need to be its own field instead of using the name and a configured github url?

Reply:
It's a remnant from when I was still using the API and it'd return the URL. If we leave things as is, I'll drop it, but it might come in handy for the groups.

Reply:
Ended up making it a property in b4911d0